packages/react/src/utils/cauldron-event.ts

```diff
@@ -0,0 +1,8 @@
+const cauldronEvent = (type: string, details?: EventInit) => {
```

Comment:
@scurker, this is a bit different than [what you suggested](https://github.com/dequelabs/cauldron/pull/161/files#r555412328)...what do you think about this util?

Reply:
Yeah, this is fine. I was just shooting off some psuedo code